src/profile/profile.ts

```diff
@@ -247,6 +247,19 @@ export class Profile extends Logger {
     }
   }
 
+  /**
+   * Adds an identity to profile.
+   *
+   * @param      {string}  address  The address
+   * @param      {string}  key      The key
+   * @param      {string}  value    The value
```

Comment:
The value for what?

Reply:
This will be the key. shall change the name.